docs/web/getting-started/basic-setup.md

```diff
@@ -171,6 +174,69 @@ initSDK({
 });
 ```
 
+In addition, initializing the SDK with `ConsoleLogRecordExporter` and `ConsoleSpanExporter` exporters allows you to take
```

Comment:
I added these that were present in our readme but not here.. Not sure if it was intentional. To me it seems that having this here is correct but let me know your thoughts.